packages/slidev/node/export.ts

```diff
@@ -106,7 +109,39 @@ export async function exportSlides({
     return url.match(/clicks=([1-9][0-9]*)/)?.[1]
   }
 
-  async function genPageWithClicks(fn: (i: number, clicks?: string) => Promise<any>, i: number, clicks?: string) {
+  async function genPagePng(pages: number[]) {
+    const genScreenshot = async(i: number, clicks?: string) => {
+      await go(i, clicks)
+      await page.screenshot({
+        omitBackground: false,
+        path: path.join(
+          output,
+          `${i.toString().padStart(2, '0')}${clicks ? `-${clicks}` : ''}.png`,
+        ),
+      })
+    }
+    for (const i of pages)
+      await genPageWithClicks(genScreenshot, i)
+  }
+
+  async function genPageMd(pages: number[], slides: SlideInfo[]) {
+    const mds: string[] = []
+
+    for (const i of pages) {
+      const mdImg = `![](./${output}/${i.toString().padStart(2, '0')}.png)\n\n`
```

Comment:
Do we also need the title of each slides here?

Reply:
title like `# Welcome to Slidev` ? 

It seems a bit redundant to force it

Reply:
How about `![${title}](${image})`? TBH I am not so confident if it could fits the general needs, as the current API there is no way to do customization.

Reply:
Yes, it looks better this way

Reply:
code updated